vllm/v1/engine/core.py

```diff
@@ -146,7 +146,7 @@ def abort_requests(self, request_ids: list[str]):
     def step(self) -> EngineCoreOutputs:
         """Schedule, execute, and make output."""
 
-        if not self.scheduler.has_unfinished_requests():
+        if not self.scheduler.has_requests():
```

Comment:
Can you please document here why we should do `has_requests` rather than `has_unfinished_requests` (like the PR description)?